### PR TITLE
Static_Pages

### PR DESCRIPTION
**Pull request by binhpt-1239**
## Related Tickets
- [#47129](https://edu-redmine.sun-asterisk.vn/issues/47129)

## WHAT (optional)
-  Create Static pages home, about, help

## HOW
- i18n, bootstrap5

## Evidence (Screenshot or Video)
Screenshot from 2022-03-15 14-24-45

Screenshot from 2022-03-15 14-25-32

### CODE REVIEW

**Comment by binhpt-1239** [comment]
ready

**Comment by BaoKhanh99** [comment]
ok

File: app/assets/stylesheets/static_pages.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (ckvvanle) [inline app/assets/stylesheets/static_pages.scss:1]: Xóa file

File: app/helpers/static_pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
+end
```
Comment (ckvvanle) [inline app/helpers/static_pages_helper.rb:2]: Xóa file

File: app/assets/stylesheets/static_pages.scss (hunk 1 above)
Comment (binhpt-1239) [reply app/assets/stylesheets/static_pages.scss]: fix

File: app/helpers/static_pages_helper.rb (hunk 2 above)
Comment (binhpt-1239) [reply app/helpers/static_pages_helper.rb]: fix

**Comment by binhpt-1239** [comment]
ready

File: app/assets/stylesheets/static_pages.scss (hunk 1 above)
Comment (binhpt-1239) [reply app/assets/stylesheets/static_pages.scss]: fix

File: app/helpers/static_pages_helper.rb (hunk 2 above)
Comment (binhpt-1239) [reply app/helpers/static_pages_helper.rb]: fix

**Pull request merged**